Menu/Menu.js

```diff
@@ -1,11 +1,23 @@
 //Using jQuery, select the menu element and the menu-button element
-const menu;
-const menuButton;
+let $menu = $(".menu");
+let $menuButton = $(".menu-button");
 
 //Create a callback that will be passed to the menuButton click event listener
 //Inside this callback a class should be either added or removed from menu
-const toggleMenu = () => {
-
+let toggleMenu = () => {
+    $(".menu").toggleClass("menu--open");
 }
 
 //Using jQuery, add a click event to the menuButton, passing it the toggleMenu function.
+$(".menu-button").click(toggleMenu);
+
+//animations
+$( ".menu-button" ).click(function() {
+    $( ".menu" ).animate({
+      opacity: .50,
+      left: "+=50",
```

Comment:
I see what you mean about the funny behavior with the menu button! I think you could use an `if` statement here to avoid this -- if the menu is visible, shift to the right, otherwise shift to the left.